app/views/candidate_mailer/submit_application_email.text.erb

```diff
@@ -33,9 +39,13 @@ You can track the progress of your <%= 'application'.pluralize(@application_form
 # Withdrawing your application
 
 <% if FeatureFlag.active?('candidate_withdrawals') %>
-  You can withdraw your application to <%= 'course'.pluralize(@application_form.application_choices.count) %> at any point, even after you’ve accepted an offer. [Sign in to your account](<%= candidate_interface_sign_in_url %>) and click ‘withdraw’ next to the <%= 'course'.pluralize(@application_form.application_choices.count) %> you want to withdraw. We’ll let your training <%= 'provider'.pluralize(@application_form.application_choices.count) %> know.
+  You can withdraw your application to your <%= 'course'.pluralize(@application_form.application_choices.count) %> at any point, even after you’ve accepted an offer.
+
+  [Sign in to your account](<%= candidate_interface_sign_in_url %>) and click ‘withdraw’ next to the <%= 'course'.pluralize(@application_form.application_choices.count) %> you want to withdraw and we’ll let your training <%= 'provider'.pluralize(@application_form.application_choices.count) %> know.
 <% else %>
-  You can withdraw your application to <%= 'course'.pluralize(@application_form.application_choices.count) %> at any point, even after you’ve accepted an offer. Contact us at [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk) if you want to withdraw and we’ll let your training <%= 'provider'.pluralize(@application_form.application_choices.count) %> know.
+  You can withdraw your application to your course(s) at any point, even after you’ve accepted an offer.
```

Comment:
Why has the pluralize been removed here?

Reply:
Eek!

Reply:
Fixed.